frontend/src/pages/CreateProfile/CurrentPfp.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import { Row, Col, Image, Form } from 'react-bootstrap'
+import Blockies from 'react-blockies';
+import ProfileEnsAndAddress from '../../components/ProfileEnsAndAddress';
+
+function CurrentPfp({ pfp, userAddress }) {
+    
+ 
+
+  
+  return (
+    <div className="mb-3">
+        {/* Profile picture */}
+        <Row className="justify-content-center text-center p-2">
+                                    <Col className="col-auto">
+                                        {pfp === null
+                                        ? <Blockies
+                                            seed={userAddress}
```

Comment:
This is throwing an error in the console for me saying userAddress is undefined, which is making the blockie show as a 404. I'll dig deeper here to see what's happening.

Reply:
Ok resolved it here
https://github.com/blocksoapp/monorepo/pull/67/commits/33884c361b4c402b7c66d4c80db5fd567d21d231

Reply:
fix: pfp === null || pfp === ''